src/rgw/CMakeLists.txt

```diff
@@ -270,6 +270,30 @@ set_target_properties(rgw PROPERTIES OUTPUT_NAME rgw VERSION 2.0.0
   SOVERSION 2)
 install(TARGETS rgw DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
+set(librgw_admin_user_srcs
+  librgw_admin_user.cc
+  rgw_admin_user.cc)
+add_library(rgw_admin_user SHARED ${librgw_admin_user_srcs})
+target_link_libraries(rgw_admin_user PRIVATE
+  rgw_a
```

Comment:
this brings in essentially everything in an instance, i.e., it's everything needed by librgw for file access;  at least the interface is small;  could this be reduced further, though?